src/assets/wise5/components/match/match-student/match-student-default/match-student-default.component.ts

```diff
@@ -383,18 +394,23 @@ export class MatchStudentDefault extends ComponentStudent {
       : $localize`Choices`;
   }
 
+  protected checkAnswer(choiceIdsExcludedFromFeedback: string[] = []): void {
+    this.checkAnswerHelper(this.buckets, choiceIdsExcludedFromFeedback);
+  }
```

Comment:
Remove need for checkAnswerHelper() by providing function param default (```buckets: any[] = this.buckets```)
```suggestion
  protected checkAnswer(
    choiceIdsExcludedFromFeedback: string[] = [],
    buckets: any[] = this.buckets
  ): void {
```